source/draw/gpu/opengl/OpenGLContext.ooc

```diff
@@ -38,13 +38,32 @@ OpenGLContext: class extends GpuContext {
 	_pointsShader: OpenGLMap
 	_meshShader: OpenGLMapMesh
 	_renderer: GLRenderer
-	_recycleBin := _RecycleBin new()
+	_recycleBinMonochrome: RecycleBin<OpenGLMonochrome>
```

Comment:
Not sure, but it's not like ooc generics have a reputation for being reliable. I can experiment later for a future PR if you want.